### ISSUE TITLE
Suggestion: make subscribeConfirmLink a config rather than a form data

### ISSUE THREAD

**Issue by cyChop** [hispanic/staticwoman#13]
For the case of double opt-in for notifications, the `subscribeConfirmUrl` must be provided as a field of `options` in the form. However, I think this link is not related to a specific comment. As such, I'm not sure it belongs in the form.

It would appear more logical to me to make it part of the site config (I know where my server is) or the server config (I know where I, the server, am).

Or is it for GDPR purpose, in order to save this information along with the rest?

**Comment by hispanic** [comment]
Good point. After revisiting the code, I imagine it could be incorporated into the JSON config schema and sourced from the server config. It was first added as part of the [GDPR-compliance pull request](https://github.com/hispanic/staticwoman/pull/6), but it isn't one of the data points saved for that purpose.
